tests/process-modeler/process-modeler.spec.ts

```diff
@@ -128,3 +128,160 @@ test('process modeler', async ({ processModelerPage }) => {
   const newSubprocessDefinitionID = processModelerPage.page.url().split('/processes/').pop();
   expect(newSubprocessDefinitionID).not.toEqual(processModelerPage.processDefinitionID);
 });
+
+test.describe('Shortcuts in Modeler', () => {
+  test('close modeler / go to process list with shortcut', async ({ processModelerPage }) => {
+    const { page } = processModelerPage;
+
+    /* Close modeler */
+    await page.getByRole('main').press('Escape');
+    await page.getByRole('main').press('Escape');
+
+    await processModelerPage.waitForHydration();
```

Comment:
What is the purpose here after the shortcuts?

Reply:
Using 'waitForHydration' allowed for using expect for the navigation / url change:
```tsx
/* Check if back at Process-List */
    await expect(page.url(), 'Could not close modeler with shortcut (2*esc)').toMatch(
      /\/processes$/,
    );
``` 

This does not work otherwise.

The test is supposed to check for a change in the navigation, which is why I put it into an expect statement. This way a descriptive failure message is displayed in case of failure.

I changed it to:
```tsx
await page.waitForURL(/\/processes$/);
``` 
But the test is not checking anything now, but rahter just applies steps.